File: LeetCode/485/485.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count = 0;
        int maxCount = INT_MIN;
        for(int i=0; i<nums.size(); i++)
        {
            if(nums[i] == 0)
            {
                maxCount = maxCount > count ? maxCount : count;
                count = 0;
            }
            else
                count++;
        }
        
        maxCount = maxCount > count ? maxCount : count;
        return maxCount;
    }
};

File: LeetCode/485/485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cur,top = 0,0
        for i in nums:
            if i == 1:
                cur +=1
            else:
                if cur > top :
                    top = cur
                cur = 0
        return max(cur,top)